src/app/components/MediaLoader/configs/tvMedia.ts

```diff
@@ -0,0 +1,49 @@
+import filterForBlockType from '#lib/utilities/blockHandlers';
+import { ConfigBuilderProps, ConfigBuilderReturnProps } from '../types';
+import buildPlaceholderConfig from '../utils/buildPlaceholderConfig';
+
+export default ({
+  blocks,
+  basePlayerConfig,
+  translations,
+}: ConfigBuilderProps): ConfigBuilderReturnProps => {
+  const { model: tvMediaBlock } = filterForBlockType(blocks, 'tvMedia');
+
+  const placeholderConfig = buildPlaceholderConfig({
+    title: tvMediaBlock.episodeTitle,
+    duration: tvMediaBlock?.versions?.[0]?.duration,
+    durationISO8601: tvMediaBlock?.versions?.[0]?.durationISO8601,
+    type: 'video',
+    holdingImageURL: `https://${tvMediaBlock.imageUrl}`,
```

Comment:
This comment may have been missed: let's also extract `holdingImageURL` into a separate variable, and use it in multiple places below.